docs/modules/configuring/con-virtualkafkacluster-mtls.adoc

```diff
@@ -0,0 +1,35 @@
+// file included in the following:
+//
+// kroxylicious-operator/assemblies/assembly-operator-secure-client-proxy-connection.adoc
+
+[id='con-kafka-client-mtls-{context}']
+= Mutual TLS configuration for client-to-proxy connections
+
+You can configure a virtual cluster ingress to request or require that the Kafka client authenticates to the proxy using TLS.
+This is known as mutual TLS (mTLS), because both the client and the proxy authenticate each other using TLS.
+
+.Example `VirtualKafkaCluster` configured so that the proxy will *require* Kafka clients to present a trusted TLS client certificate.
```

Comment:
```suggestion
.Example `VirtualKafkaCluster` configuration requiring clients to present a trusted certificate
```

Reply:
Not sure we need VirtualKafkaCluster in the title, so we could make it shorter by removing, but we should be consistent with approach in doc